{"publishedDate": "2020-06-11T02:15Z", "lastModifiedDate": "2024-11-21T05:02Z", "cve": {"data_type": "CVE", "data_format": "MITRE", "data_version": "1.0", "CVE_data_meta": {"ID": "CVE-2020-13998", "ASSIGNER": "cve@mitre.org"}, "description": {"description_data": [{"lang": "en", "value": "Citrix XenApp 6.5, when 2FA is enabled, allows a remote unauthenticated attacker to ascertain whether a user exists on the server, because the 2FA error page only occurs after a valid username is entered. NOTE: This vulnerability only affects products that are no longer supported by the maintainer"}, {"lang": "es", "value": "** VERSI\u00d3N NO COMPATIBLE CUANDO SE ASIGN\u00d3 ** Citrix XenApp versi\u00f3n 6.5, cuando 2FA est\u00e1 habilitado, permite a un atacante remoto no autenticado determinar si existe un usuario en el servidor, porque la p\u00e1gina de error 2FA solo se produce despu\u00e9s de ingresar un nombre de usuario v\u00e1lido. NOTA: Esta vulnerabilidad s\u00f3lo afecta a los productos que ya no son apoyados por el mantenedor"}]}, "references": {"reference_data": [{"url": "https://gist.github.com/kampji/11e259d68ad98a6f0f898132f1961a96", "name": "", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "https://gist.github.com/kampji/11e259d68ad98a6f0f898132f1961a96", "name": "", "refsource": "", "tags": ["Third Party Advisory"]}]}, "problemtype": {"problemtype_data": [{"description": [{"lang": "en", "value": "CWE-203"}, {"lang": "en", "value": "CWE-639"}]}]}}, "impact": {"baseMetricV2": {"cvssV2": {"version": "2.0", "vectorString": "AV:N/AC:M/Au:N/C:P/I:N/A:N", "baseScore": 4.3, "accessVector": "NETWORK", "accessComplexity": "MEDIUM", "authentication": "NONE", "confidentialityImpact": "PARTIAL", "integrityImpact": "NONE", "availabilityImpact": "NONE"}, "exploitabilityScore": 8.6, "impactScore": 2.9, "severity": "MEDIUM", "obtainAllPrivilege": false, "obtainUserPrivilege": false, "obtainOtherPrivilege": false, "userInteractionRequired": false}, "baseMetricV3": {"exploitabilityScore": 3.9, "impactScore": 1.4, "cvssV3": {"version": "3.1", "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N", "baseScore": 5.3, "baseSeverity": "MEDIUM", "attackVector": "NETWORK", "attackComplexity": "LOW", "privilegesRequired": "NONE", "userInteraction": "NONE", "scope": "UNCHANGED", "confidentialityImpact": "LOW", "integrityImpact": "NONE", "availabilityImpact": "NONE"}}}, "configurations": {"CVE_data_version": "4.0", "nodes": [{"operator": "OR", "negate": false, "children": [], "cpe_match": [{"vulnerable": true, "cpe23Uri": "cpe:2.3:a:citrix:xenapp:6.5.0.0:*:*:*:*:*:*:*", "matchCriteriaId": "C455A80D-45E2-4963-981D-610D9C24391C", "cpe_name": []}]}]}}